### PR TITLE
fix for #504

### PR DESCRIPTION
**Pull request by fabianfrz**

### CODE REVIEW

File: src/www/firewall_rules.php [hunk 1]
```diff
@@ -617,6 +624,8 @@
                           <td class="nowrap"><?=gettext("log (disabled)");?></td>
                           <td width="16"><span class="fa fa-long-arrow-left"></span></td>
                           <td width="100"><?=gettext("out");?></td>
+                          <td width="16"></td>
+                          <td width="100"></td>
```
Comment (fichtner) [inline src/www/firewall_rules.php]: this misses a grey flash icon and the description "match last"

File: src/www/firewall_rules.php [hunk 2]
```diff
@@ -599,6 +604,8 @@
                           <td width="100"><?=gettext("log");?></td>
                           <td width="16"><span class="fa fa-long-arrow-right"></span></td>
                           <td width="100"><?=gettext("in");?></td>
+                          <td width="16"><span class="fa fa-flash"></span></td>
+                          <td width="100"><?=gettext("quick");?></td>
```
Comment (fichtner) [inline src/www/firewall_rules.php]: fa fa-flash text-warning + "match first"

**Comment by fabianfrz** [comment]
@fichtner applied changes

**Comment by fichtner** [comment]
Thanks, will tweak a tiny bit :)

**Pull request merged**